### ISSUE TITLE
Include model is not executing query from cache

### ISSUE THREAD

**Issue by SohaibFarooqi** [rfink/sequelize-redis-cache#15]
Hi,
I tried following query:
```javascript
var cacheObj = models.cacher('type').ttl(5000);
return cacheObj.findAll
                  (
                       {
                          include   : [{model: models.type_lang , attributes: ['type_title_l1',                      
                                                      'type_alternate_title_l1']}],
                           attributes: ['type_id' , 'type_title' , 'type_htaccess' , 'p' , 'type_alternate_title']
                        }
                   ).then(function(all_types) {
                                          console.log(cacheObj.cacheHit); // true or false
                                          return all_types;
                                   });
                                 ```
But everytime it shows false on console.log(cacheObj.cacheHit). What have i missed?

**Comment by rfink** [comment]
Hello and thanks for trying out my module,

Can you turn this into a plunker or jsfiddle?  I'd be happy to take a look.

**Comment by NotJustClarkKent** [comment]
I am having the same issue, taking away the `includes:` property from `find` (findOne/findAll) and the object is returned from the cache.

**Comment by rfink** [comment]
Thanks for the fix!  I've published as version 1.2.0, along with dependency updates so this module can be tested with node 4.x, 5.x, and 6.x.

**Comment by NotJustClarkKent** [comment]
Glad I could help, thanks for your work and the quick turnaround on that PR! 💯
